### ISSUE TITLE
In Leiningen, setting :resource-paths to include "target" is bad for uberjars

### ISSUE THREAD

**Issue by olieidel** [bhauman/figwheel-main#134]
Hey Bruce,

First off, thanks for all the work you put into figwheel & figwheel-main. They're awesome! I've been a long-time figwheel user and have been trying out figwheel-main over the holidays (tldr: super happy and impressed).

---

I've been working on a clj+cljs project using Leiningen as build tool.
For the Leiningen setup, the README states (`project.clj`):
```
 ;; setup target as a resource path
:resource-paths ["target" "resources"]
```

While this is necessary for development (the js files end up in `target/` per the default settings), this seems to lead to very bad behaviour when compiling an uberjar.
In my understanding, Leiningen tries to include all "resources" (usually the `resources/` directory) in the jar file. With the config above, Leiningen will also include the contents of the `target/` directory. However, the generated jar file is also written to `target/`!

At least on my machine this leads to some sort of "infinite jar-generation" in which the jar file grows endlessly larger while the uberjar compilation doesn't complete. Not including `target` in `:resource-paths` "fixes" it.

Possible solutions:

1. I'm doing something wrong (quite possible)
2. Don't include `target/` in `:resource-path`, manually set the `:output-path` to something else (e.g., `resources/dev/` and update the README accordingly. Possibly change the default behaviour of figwheel-main to do this.

**Comment by olieidel** [comment]
Yep, that's how I solved it. Should I edit the README accordingly and create a small PR?

**Comment by bhauman** [comment]
Yes thank you that would be super helpful
